webpack/assets/javascripts/react_app/components/common/Slot/Slot.js

```diff
@@ -0,0 +1,44 @@
+import { cloneElement, isValidElement } from 'react';
+import PropTypes from 'prop-types';
+import { selectMaxComponent, selectComponentByWeight } from './SlotSelectors';
+
+const Slot = ({ fills, id, multi, children, ...props }) => {
+  const addProps = object => {
+    if (multi && !object.key) {
+      throw new Error('Please add a key attriubte to multiple fills');
+    }
+
+    if (isValidElement(object)) {
+      return cloneElement(object, { ...props });
+    }
+
+    if (children === undefined) {
+      throw new Error('Slot with override props must have a child');
+    }
+
+    return cloneElement(children, { ...props, ...object });
+  };
+
+  if (Object.keys(fills).length) {
+    if (multi) {
+      return selectComponentByWeight(id).map(component => addProps(component));
+    }
+    return addProps(selectMaxComponent(id));
+  }
+  if (children) return children;
+  return null;
```

Comment:
Can be `return children || null`.
Or `children` default value can be set to `null`